src/commands/go-test.yaml

```diff
@@ -33,10 +38,10 @@ steps:
         echo -n "-w -linkmode 'auto' -extldflags '-static'" > .ldflags
 
         # buildTimestamp
-        echo -n " -X '$(go list .)/pkg/project.buildTimestamp=$(date --utc '+%FT%TZ')'" >> .ldflags
+        echo -n " -X '$(go list <<parameters.path>>)/pkg/project.buildTimestamp=$(date --utc '+%FT%TZ')'" >> .ldflags
```

Comment:
I would recommend quoting these like: `$(go list "<<parameters.path>>")`, otherwise it could be abused to execute arbitrary code in the pipelines, lets say when the path is set to: `.; echo hello; execute harmful code`. Same below.

Reply:
Good point. I fixed that.

Tested and working :heavy_check_mark: 

![image](https://github.com/user-attachments/assets/1072d620-30ae-4c67-890d-0333101dad32)
https://app.circleci.com/pipelines/github/giantswarm/grafana-multi-tenant-proxy/516/workflows/d7b2853a-813e-493c-83bf-60547aa057c4/jobs/994